### ISSUE TITLE
A more dynamic text for the pull request summary

### ISSUE THREAD

**Issue by Konard** [irgolic/AutoPR#83]
![Screenshot_20230505_010352](https://user-images.githubusercontent.com/1431904/236339675-01a87323-48f1-4b5e-8354-f5157eb86747.png)

This block is always collapsed when PR text is updated.
So I suggest to make it more dynamic by adding last step that is being executed in parentheses.

Instead of:
```
Planning pull request...
```
Write
```
Planning pull request... (Current step: Ran raw query)
```

**Comment by irgolic** [comment]
Should we just keep the last one open by default instead?

**Comment by irgolic** [comment]
This was fixed by #102
